desktop_version/src/Game.cpp

```diff
@@ -124,6 +124,7 @@ void Game::init(void)
     edteleportent = 0; //Added in the port!
     companion = 0;
     roomchange = false;
+    isingamecompletescreen = false;
```

Comment:
Instead of introducing another variable, you can probably just check for `game.state` instead. That way, if the Game Complete gamestate sequence gets interrupted, the timer won't go missing.

Reply:
Alright, gonna try to figure out all the game complete states so it doesn't end up flashing on and off

Reply:
Ok, well, I'll help you out. It should just be sufficient to do `(game.state >= 3501 && game.state <= 3518) || (game.state >= 3520 && game.state <= 3522)`.

Reply:
that should work yeah

```
bool Game::isingamecompletescreen()
{
    return (game.state >= 3501 && game.state <= 3518) || (game.state >= 3520 && game.state <= 3522);
}
```

do you think having it as a class method is overkill?

Reply:
Not overkill at all - I do the same elsewhere, e.g. `incompetitive()`, `inspecial()`.

Reply:
Done :+1: